src/rdpq/rdpq_tex.c

```diff
@@ -15,9 +15,14 @@
 #include <math.h>
 
 /** @brief Non-zero if we are doing a multi-texture upload */
-static int multi_upload = 0;
-static int multi_upload_bytes = 0;
-static int multi_upload_limit = 0;
+typedef struct rdpq_multi_upload_s {
+    int  used;
+    int  bytes;
+    int  limit;
+
+    tex_loader_t last_tload;
```

Comment:
since we're using this also outside of the multi sequence, maybe we should also move this outside of rdpq_multi_upload_t, for clarity. I'm not super excited about always using a global instance of texloader rather than a stack instance like before, but I guess we can live with this for a while.